test/svm/SvmSpoke.Bundle.ts

```diff
@@ -206,7 +206,7 @@ describe("svm_spoke.bundle", () => {
       .rpc();
 
     // Verify the ExecutedRelayerRefundRoot event
-    await new Promise((resolve) => setTimeout(resolve, 500)); // Wait for event processing
+    await new Promise((resolve) => setTimeout(resolve, 1000)); // Wait for event processing
```

Comment:
Were the tests failing with a 500ms delay?

Hopefully, we will find a good alternative to these sleep functions.